# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/labplot/hotfix-qt-guiprivate.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- labplot-2.12.1/CMakeLists.txt.vanilla	2026-02-08 17:51:53.541297317 +0100
+++ labplot-2.12.1/CMakeLists.txt	2026-02-08 17:55:16.790180294 +0100
@@ -50,9 +50,6 @@
     Test
 )
 
-# building QADS or Xlsx requires Qt${QT_VERSION_MAJOR}GuiPrivate (QADS is required, Xlsx is optional)
-find_package(Qt${QT_MAJOR_VERSION}Gui ${QT_MIN_VERSION} CONFIG REQUIRED Private)
-
 find_package(KF${KF_MAJOR_VERSION} ${KF_MIN_VERSION} REQUIRED COMPONENTS
     Archive
     Completion
